backend/src/Hooks/Actions/OnEnqueueScripts.php

```diff
@@ -124,6 +133,7 @@ public function generate_frontend_config() {
 			'pluginURL'         => FL_ASSISTANT_URL,
 			'taxonomies'        => $this->posts->get_taxononies(),
 			'userRoles'         => $this->users->get_roles(),
+			'sysCommentStatus'  => $system_comment_status,
```

Comment:
@shahidajmeri786 We use the term `system` a lot in Assistant. Can we change this to `wpCommentStatus` so it's not confused with something coming from the Assistant system?

Reply:
Updated the variable name to 'wpCommentStatus' and made changes.